addons/message_try_firefox_extension/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "api_version": "0.1",
+  "id": "message_try_firefox_extension",
+  "name": "Try the Firefox extension",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.25.1",
+    "enabled_features": ["webExtension"],
```

Comment:
++`localProxy`

Reply:
Actually, we just removed the `localProxy` feature in PR #10058 since it's no longer something that the client can toggle (it's just a service that happens to exist on Windows and Linux). The "feature" still gets reported through the web extension, but it's value just checks if the corresponding serivce is running.

Reply:
Hmm, maybe that was a mistake and I should put it back as an un-toggle-able feature.

Reply:
@lesleyjanenorton , were you suggesting adding `localProxy` in addition to `webExtension`?

(I'm looking for the name of a feature that will be the thing we "flip on" to make this go live for users - to be clear this is not a user-toggleable thing.)

Reply:
To summarize a zoom discussion: I'm not in favor of adding a `localProxy` feature flag back. The proxy, being a windows/systemd service required administrative privileges to start and stop and the client UI does not hold such privileges... to make this controllable by a feature flag it would have to be managed through the daemon and that is a lot of work for not a lot of gain.

Reply:
Updated per our meeting